clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesStatefulSetHandler.java

```diff
@@ -97,9 +97,9 @@ protected KubernetesCachingAgentFactory cachingAgentFactory() {
 
   @Override
   public Status status(KubernetesManifest manifest) {
-    V1beta2StatefulSet v1beta2StatefulSet =
-        KubernetesCacheDataConverter.getResource(manifest, V1beta2StatefulSet.class);
-    return status(v1beta2StatefulSet);
+    V1StatefulSet v1StatefulSet =
```

Comment:
Also here please introduce the `SUPPORTED_API_VERSIONS` collection with `apps/v1` as the only entry, like in `Deployment`, `ReplicaSet` and `DaemonSet` handlers.